lib/king_konf/decoder.rb

```diff
@@ -17,6 +17,10 @@ def string(value, **)
       value
     end
 
+    def symbol(value, **)
+      value
```

Comment:
Doesn't this need to do `value.to_sym`?

Reply:
true. :man_facepalming: will adapt specs and fix this.